spec/models/rent_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Rent, type: :model do
```

Comment:
I don't remember if i noted this on the other pr's, but we try to avoid using the `RSpec.` prefix, since it's not needed

Reply:
The other PR's are ok. This spec was auto generated from the rails command and I didn't realize it has the Rspec prefix.